packages/api-clients/package.json

```diff
@@ -29,7 +29,7 @@
     "@types/qs": "6.9.15",
     "openapi-types": "12.1.3",
     "handlebars": "4.7.7",
-    "openapi-zod-client": "1.18.1",
+    "openapi-zod-client": "1.18.3",
```

Comment:
This update of `openapi-zod-client` is cherry-picked from #1885 to allow me to test uuid validation in arrays